README.md

```diff
@@ -1,37 +1,95 @@
-# Slashtags
+<p align="center">
+  <a href="https://github.com/synonymdev/slashtags" title="Slashtags">
+    <img alt="slashtags" src="./docs/_img/slashtags-logo.png" width="150"></img>
+  </a>
+</p>
 
-Slashtags is set of methods and protocols for utilizing keypairs to establish accounts, contacts, and abstracted endpoints to create cross-authenticated communication channels, verifiable attestations & metadata, and ultimately, private-yet-interoperable web-of-trust networks.
+<h3 align="center">JavaScript implementation of the Slashtags protocol</h3>
 
-Slashtags currently supports a DID format for establishing self-sovereign identities, and in the future it will support personal data stores along with related access control features.
+---
 
-From these primitives, we will demonstrate how users can appy Slashtags to do many powerful things within web and Bitcoin applications by providing modules, demos, full applications, servers and web platforms that utilize Slashtags, Hypercore, Bitcoin, and the Lightning Network together.
+**⚠️ This is still in alpha. Please use at your own risk.⚠️**
 
-This is a monorepo containing Javascript packages for various Slashtags modules.
+---
 
-> A practical and gradual exploration of Slashtags protocol design, through a mvp implementation.
+## Overview
 
-## ⚠️ Warning
+Slashtags aims to enable self-sovereign identity, reputation and web-of-trust on the web.
 
-This is pre-alpha software. Please use at your own risk. Expect breaking changes on minor releases.
+### The problem
 
-## Usage
+IPs (Internet Protocol address) identifies where your machine exists both geographically and topologically. But you yourself (or your machine for that matter) can't be identified or addressed regardless of its location or the network it is using.
 
-### Getting started
+So the Web became a network of mediators and service providers that you have to be tethered to one of them to be addressed or identified. Giving rise to censorship, surveillance and vendor-lock problems the latest of which slowed down the rate of innovation significantly.
 
-Currently the best way to familiarize yourself with existing features is to try these [Demos](./demo/README.md).
+### Slashtag's solution
 
-If you rather see it in action before diving into the code, try the [Slashtags wallet demo](slash-wallet.netlify.app) that is already built and deployed.
+Slashtags solution is to use keyPairs to identify and address peers and attach metadata about them, giving rise to reputation, and private yet interoperable webs-of-trust.
```

Comment:
I think this is good brief explanation but it doesn't obviously match up to the stated problem.

The stated problem is that, "we have problem with censorship, surveillance, vendor lock-in."

While the solution description doesn't exactly say how Slashtag prevents those things from happening. I think maybe expanding, or giving examples of how ST solves those specific problems would be an improvement .

Reply:
My attempt at it 
```
Slashtags solution is to use keyPairs to identify and address peers and attach metadata about them. The combination of these  simple concepts would be enough abstract away walled gardens, build P2P apps and allow people to own their identity, and give rise to web of trust, interoperable reputation systems many other use cases.
"Not your keys, Not your identity"
```